README.md

```diff
@@ -19,14 +19,17 @@
 #
 -->
 
-[![Build Status](https://travis-ci.org/apache/cordova-node-xcode.svg?branch=master)](https://travis-ci.org/apache/cordova-node-xcode)
+# cordova-node-xcode
 
-# node-xcode
+[![NPM](https://nodei.co/npm/xcode.png?compact=true)](https://nodei.co/npm/xcode/)
+[![Build Status](https://travis-ci.org/apache/cordova-node-xcode.svg?branch=master)](https://travis-ci.org/apache/cordova-node-xcode)
 
-> parser/toolkit for xcodeproj project files
+Parser toolkit for xcodeproj project files
 
 Allows you to edit xcodeproject files and write them back out.
 
+based on [alunny / node-xcode](https://github.com/alunny/node-xcode)
```

Comment:
```suggestion
based on donated code [alunny/node-xcode](https://github.com/alunny/node-xcode)
```

Reply:
suggestion applied with one more word added, hope it is ok now